python/seisma/client.py

```diff
@@ -211,6 +211,23 @@ def add_case_to_job(self, name, description=None):
 
         return self.post(url, json=json)
 
+
+    @will_expected(200)
+    def update_case_in_job(self, name, description=None):
+        url = '{}{}'.format(
+            self.api_url,
+            '/jobs/{}/cases/{}'.format(
+                self.job_name, quote(name),
+            ),
+        )
+        json = {}
+
+        if description:
```

Comment:
according to https://github.com/seisma-test-analytics/seisma-api/blob/8a643435e8daffd32601e5871242f666271c5cfc/seisma/api/schemas/v1/case.post_put.json `put` will fail without description. So it should be obligatory to send here, or optionary to receive in API.

Reply:
Description are  optionary to receive in API, because it is not defined as "required" in schema.

Reply:
Made corrections based on comments. 
Now the put request always contains all parameters, and they are strictly validated according to the scheme.